spec/models/binder_spec.rb

```diff
@@ -1,5 +1,70 @@
 require 'rails_helper'
 
 RSpec.describe Binder, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe "validates binder attributes" do
+    it "quantity is present" do
+      user1 = User.create(email:"email@email.com",password:"password",username:"user1")
+      card1 = Card.create(
```

Comment:
if you find yourself wanting to reduce code duplication, this kind of logic is actually the code that can live in your helper_spec.rb files. If you're interested in the refactor, this might be a helpful resource: https://relishapp.com/rspec/rspec-rails/v/2-0/docs/helper-specs/helper-spec